src/main/java/racingcar/Validator.java

```diff
@@ -0,0 +1,54 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class Validator {
+    public static int InputNumValidator(String intInput) {
+        if (intInput.trim().isEmpty()) {
+            throw new IllegalArgumentException("입력이 잘못되었습니다.");
+        }
+
+        int number;
+
+        try {
+            number = Integer.parseInt(intInput);
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException("입력이 잘못되었습니다.");
+        }
+
+        return number;
```

Comment:
이러면 굳이 블록 밖에서 변수를 선언할 필요 없겠네요~

```suggestion
        try {
            return Integer.parseInt(intInput);
        } catch (NumberFormatException e) {
            throw new IllegalArgumentException("입력이 잘못되었습니다.");
        }
```

Reply:
헉 맞네요...! 실수한 것 같습니다..😥